lab-ron/lib/greet.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
```

Comment:
This works but doesn't conform exactly to the assignment specifications, please see my comment below.